server/app/jobs/certificate_renew_job.rb

```diff
@@ -0,0 +1,91 @@
+class CertificateRenewJob
+  include Celluloid
+  include Logging
+  include CurrentLeader
+  include WaitHelper
+  include DistributedLocks
+
+  RENEW_INTERVAL = 1.day.to_i
```

Comment:
Re-commenting on this because I wrote it in the wrong place, and it got lost: Wondering if the `RENEW_INTERVAL = 1.day` is now even too long... it means that if the server gets restarted every day, this job might never actually run?

I think I'd stick to maximum delays on the order of an hour for timers like `every`/`sleep`... longer intervals would really need some kind of scheduling system that manages the timestamps in mongo?

So maybe drop the RENEW_INTERVAL down to like 1h?

Reply:
Correction, this is less of a concern now with the initial 5-minute run. I'm a little wary of doing a 24h `Celluloid.sleep`, but I don't see why it wouldn't work.

Long-term we might want to have some kind of mongo-based scheduled tasks thing.